### DISCUSSION TITLE
GraphConv Equation

### DISCUSSION THREAD

**Discussion by anthonysirico** [pyg-team/pytorch_geometric#5306] [Q&A] [answered]
i am trying to better understand the equation found here https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.conv.GraphConv under GraphConv. Can anyone tell me what each variable represents? I looked the referenced paper and cannot find what x or x' would represent. Is that the feature matrix?

**Comment by rusty1s** [comment, marked as answer]
The notation follows the standard PyG semantic introduced in the message passing tutorial. `x` refers to the input features of a node, and `x‘` refers to the output features.

**Reply by anthonysirico** [reply]
I am assuming that W is the weights that are adjusted after each iteration?

**Reply by rusty1s** [reply]
Yes, that is correct.

**Reply by anthonysirico** [reply]
Thank you.
